### DISCUSSION TITLE
Release Sketchware Pro

### DISCUSSION THREAD

**Discussion by Jeiel0rbit** [Sketchware-Pro/Sketchware-Pro#1044] [General]
I ask you to compile the code to ".apk". It hasn't been updated in over a year. I also notice that there are problems with API higher than 11. Unfortunately, the Android 13 that I currently use is flawed in some cases. This has also been discussed before. If anyone has ".apk" of Sketchware Pro compiled, please send me.

**Comment by j-px** [comment]
[Use a nightly build as described here](https://github.com/Sketchware-Pro/Sketchware-Pro/discussions/931#discussioncomment-6574106). There'll be a release when we've got the time to create one and feel that it's stable enough.

**Reply by Jeiel0rbit** [reply]
Hm... Ok. I will wait then.
